contracts/CryptoMafiaCoin.sol

```diff
@@ -4,12 +4,45 @@ pragma solidity ^0.8.0;
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
+interface RandomInterface {
+  function getRandomNumber(uint256 userProvidedSeed) external returns (bytes32);
+  function randomNumber() external view returns(uint256);
+}
+
+interface AccountInterface{
+    function ownerOf(uint256 tokenId) external view returns (address);
+}
+
 contract CryptoMafiaCoin is ERC20, Ownable {
     address private _accountsHolder;
-
-    constructor(address accountsHolder, uint256 initialSupply) ERC20("Crypto Mafia Coin", "CMC") {
+    
+    address addressRandomGenerator;
+    
+    address contractAddress;
+    
+    RandomInterface RNG;
+    
+    AccountInterface AI;
+    
+    mapping(uint256=>uint256) roomStake;
```

Comment:
What's the need for this?

Reply:
This tells us the total amount staked in the room for a given roomId